cogito/put.go

```diff
@@ -16,7 +16,7 @@ type Putter interface {
 	// ProcessInputDir validates and extract the needed information from the "put input".
 	ProcessInputDir() error
 	// Sinks return the list of configured sinks.
-	Sinks() []Sinker
+	Sinks() ([]Sinker, error)
```

Comment:
The `Sinks()` method now return also an error. If for example the sink is mispelled or simply not supported it will be reported.